signinapp/auth.py

```diff
@@ -50,10 +60,18 @@ class ChangePasswordForm(FlaskForm):
 @auth.route("/register", methods=["GET", "POST"])
 def register():
     form = RegisterForm()
+    form.subteam.choices = Subteam.get_subteams()
     if form.validate_on_submit():
         # code to validate and add user to database goes here
         name = form.name.data
         password = form.password.data
+        preferred_name = form.preferred_name.data
```

Comment:
Thinking about it, can we move this block to after the check for a user already existing?